CorePreferences/src/au/gov/asd/tac/constellation/preferences/ApplicationPreferenceKeys.java

```diff
@@ -188,9 +188,25 @@ public static String getRESTDir(final Preferences prefs) {
      */
     public static final String FONT_PREFERENCES = "au/gov/asd/tac/constellation/preferences";
     public static final String FONT_FAMILY = "fontFamily";
-    public static String FONT_FAMILY_DEFAULT = "Arial";
     public static final String FONT_SIZE = "fontSize";
-    public static String FONT_SIZE_DEFAULT = "12";
+    private static String fontFamilyDefault = "Arial";
+    private static String fontSizeDefault = "12";
```

Comment:
I think when I look at this change, my gut questions why the default values need to be changed at all? I can see in code that they get changed (hence they're not final) but given all other preference keys don't ever modify the default value I'm  curious to know the need to alter it for these 2.

What I'm trying to get at is that if the defaults have to be edited then what you have here is fine, but if we can refactor code such that the defaults can be made final, that would probably be my preferred solution here

Reply:
Managed to refactor. For some reason the values were being set on startup...to themselves. So that got removed.